config/opal_setup_cc.m4

```diff
@@ -365,7 +365,7 @@ AC_DEFUN([OPAL_SETUP_CC],[
         [opal_cv_cc_supports___builtin_prefetch],
         [AC_LINK_IFELSE([AC_LANG_PROGRAM([],
           [int ptr;
-           __builtin_prefetch(&ptr,0,0);])],
+           int __builtin_prefetch(&ptr,0,0);])],
```

Comment:
Can you clarify what this change is for?  From my reading, it changes the body of `main()` in the test from:
```c
int ptr;
__builtin_prefetch(&ptr,0,0);
```
to
```c
int ptr;
int __builtin_prefetch(&ptr,0,0);
```
which doesn't appear to be valid C.

Reply:
Sorry, that's my fault for hurrying. That's definitely dodgy. I got distracted doing something else (I'll file a separate PR for that) and was a bit autopilot by then.

But apparently I can't reproduce this bit anyway, sorry for the noise.